server/monitor/src/main/java/org/apache/accumulo/monitor/rest/gc/GarbageCollectorStats.java

```diff
@@ -52,6 +52,6 @@ public GarbageCollectorStats(String type, GcCycleStats thriftStats) {
     this.inUse = thriftStats.inUse;
     this.deleted = thriftStats.deleted;
     this.errors = thriftStats.errors;
-    this.duration = this.finished - thriftStats.started;
+    this.duration = thriftStats.finished - thriftStats.started;
```

Comment:
This change isn't a functional one... it just makes it more clear that they're both coming from the same source when computing the difference.